futures/failureaccess/pom.xml

```diff
@@ -5,11 +5,12 @@
   <parent>
     <groupId>com.google.guava</groupId>
     <artifactId>guava-parent</artifactId>
-    <version>26.0-android</version>
+    <version>1.0-HEAD-jre-SNAPSHOT</version>
+    <relativePath>../../pom.xml</relativePath>
   </parent>
   <artifactId>failureaccess</artifactId>
-  <version>1.0.2</version>
-  <packaging>bundle</packaging>
+  <version>1.0.3-jpms</version>
+  <packaging>jar</packaging>
```

Comment:
Exactly one more release will need to take place on `failureaccess`. This new artifact version contains a `module-info` definition which exports the classes needed for Guava.